src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -283,7 +283,7 @@ public class InternalController implements IInternalController, ITwoToOne, IBtoA
      */
     private AutoStarter autoStarter;
 
-    private AutoStopContestClockThread autoStopContestClockThread = null;
+    private ContestClockMonitorThread autoStopContestClockThread = null;
```

Comment:
I sort of feel like if the name of the CLASS was changed (a good change), it would have been better to also change the name of the _variable_  (`autoStopContestClockThread`).  Otherwise, down the road, the name `autoStopContestClockThread` is going to cause confusion...

Reply:
Good point.  I will change that.  In the last push before merge.